products/jbrowse-desktop/package.json

```diff
@@ -144,5 +146,13 @@
     },
     "afterSign": "scripts/notarize.js"
   },
-  "private": true
+  "private": true,
+  "protocols": [
+    {
+      "name": "jbrowse-deep-linking",
+      "schemes": [
+        "jbrowse"
+      ]
+    }
+  ]
```

Comment:
Does `protocols` not need to be inside the `build` section? I ask because that's where the rest of the electron build config is.

Reply:
I followed the tutorial and did it accordingly. Even in linux the custom protocol was registered successfully but I think upon building it might not work. I will check it once.